common/changes/@microsoft/rush/removeRushLink_2020-07-14-01-11.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Remove support for rush-link.json",
```

Comment:
This is pretty vague. Could you explain to an end user what changed and how it affects them? Also the wording makes it sound like you made Rush worse :-P

Reply:
iPhone app changelogs have me jaded. "Bug fixes" :P